tests/phpunit/tests/blocks/register.php

```diff
@@ -367,6 +367,44 @@ function test_block_registers_with_metadata_fixture() {
 		$this->assertSame( 'my-plugin-notice-style', $result->style );
 	}
 
+	/**
+	 * @ticket 000
+	 */
+	function test_block_registers_with_metadata_i18n_support() {
```

Comment:
@swissspidy or @ocean90, is that fine to reuse existing translations for the test plugin here?

The goal of the test is to ensure that those translations are applied properly after data is loaded from `block.json`.

Reply:
Might be better to use a specific one for this case. One that doesn't use the same string for different attributes like `title` and `description` should be different.

Reply:
Might be better to use a specific one for this case. One that doesn't use the same string for different attributes like `title` and `description` should be different.

Reply:
Yes, @swissspidy had the same comment. I consider reusing the existing `block.json` fixture. The only challenge for me is how do I generate files with translations. Do I need to run a WP-CLI command on https://github.com/WordPress/wordpress-develop/blob/master/tests/phpunit/tests/blocks/fixtures/block.json or do I need to run it on the plugin that contains this file? I would appreciate some pointers.

Reply:
Poedit did the trick, I don't know if it was the best way but 🤷🏻